postgres/Dockerfile

```diff
@@ -0,0 +1,33 @@
+# Linux for Health PostgreSQL image.
+#
+# This image serves PostgreSQL within a Red Hat UBI container.
+# Sample PostgreSQL Dockerfile: https://docs.docker.com/engine/examples/postgresql_service/
+#
+# Environment variables:
+# - PGDATA: The PostgreSQL data directory.
+
+FROM registry.access.redhat.com/ubi8/ubi-init:latest
+
+ENV PGDATA=/var/lib/pgsql/data/
+
+# Install PostgreSQL 12
```

Comment:
It would be worth taking a look to see if we can support the rpm URL and "package name" as build arguments within the Dockerfile. This approach would allow use to reuse this image for future pg versions and is similar to how we are working with the [OpenJDK](https://github.com/LinuxForHealth/images/blob/master/openjdk/Dockerfile#L24)

For example:
```
ARG POSTGRES_RELEASE_URL=https://download.postgresql.org/pub/repos/yum/reporpms/EL-8-x86_64/pgdg-redhat-repo-latest.noarch.rpm
ARG POSTGRES_PACKAGE=postgresql12-server

RUN dnf -y install ${POSTGRES_RELEASE_URL} I && \
         ${POSTGRES_PACKAGE} && dnf clean all
```
For the latter RUN command, we may be able to compress the install and cleanup into a single "layer"